newsroom/notifications/utils.py

```diff
@@ -59,6 +58,7 @@ async def get_notifications_with_items() -> dict[str, Any] | None:
     """
 
     from newsroom.agenda import AgendaSearchServiceAsync
+    from newsroom.wire import WireSearchServiceAsync
```

Comment:
Instead of moving the import, can you please import the type at the top of this file, for both Agenda and Wire, and use the [find_by_ids](https://superdesk.readthedocs.io/en/async/core/resource_management.html#superdesk.core.resources.service.AsyncResourceService.find_by_ids) method